src/main/rx/spektrum.c

```diff
@@ -210,52 +207,80 @@ bool spekShouldBind(uint8_t spektrum_sat_bind)
  */
 void spektrumBind(rxConfig_t *rxConfig)
 {
-    int i;
     if (!spekShouldBind(rxConfig->spektrum_sat_bind)) {
         return;
     }
 
-    LED1_ON;
+    ioTag_t bindPin;
+
+    if (rxConfig->spektrum_bind_pin_override_ioTag) {
+        bindPin = rxConfig->spektrum_bind_pin_override_ioTag;
+    } else {
+        const serialPortConfig_t *portConfig = findSerialPortConfig(FUNCTION_RX_SERIAL);
+        if (!portConfig) {
+            return;
+        }
 
-    BindPin = IOGetByTag(IO_TAG(SPEKTRUM_BIND_PIN));
-    IOInit(BindPin, OWNER_RX_BIND, 0);
-    IOConfigGPIO(BindPin, IOCFG_OUT_PP);
+        int index = SERIAL_PORT_IDENTIFIER_TO_INDEX(portConfig->identifier);
+        ioTag_t txPin = serialPinConfig()->ioTagTx[index];
+        ioTag_t rxPin = serialPinConfig()->ioTagRx[index];
+
+        switch (rxConfig->serialrx_provider) {
+        case SERIALRX_SRXL:
+#ifdef TELEMETRY
```

Comment:
I think the conditional on TELEMETRY can be removed.

The code section was stolen from `spektrumInit`, and I believe the conditional is there to save flash.